lib/CMakeLists.txt

```diff
@@ -82,6 +82,11 @@ if (FL_LIBRARIES_USE_CUDA)
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/..> # TODO(jacobkahn): fix me
     )
 
+  # TODO For CUDA version 11 CUB doesn't have to be built 
+  if (CUDA_VERSION VERSION_GREATER "11.0" OR CUDA_VERSION VERSION_EQUAL "11.0")
```

Comment:
Could you potentially gate https://github.com/facebookresearch/flashlight/blob/cc7f8073b719f1d7aba289ba0e50ec9e94dca5db/lib/CMakeLists.txt#L55 with this check as well so we don't download Cub if we've found CUDA 11?

Reply:
I've tried to do that, but it didn't work, and I didn't have the time to dig into it.

Reply:
Also, the latest version of [FindCudaToolkit](https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html#module:FindCUDAToolkit) (which should be updated in flashlight CMakeLists.txt) doesn't compile *cub* library yet, so we need to do it by hand.